lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -478,7 +478,8 @@ bool getSPIRVBuiltin(const std::string &OrigName, spv::BuiltIn &B) {
   SmallVector<StringRef, 2> Postfix;
   StringRef R(OrigName);
   R = dePrefixSPIRVName(R, Postfix);
-  assert(Postfix.empty() && "Invalid SPIR-V builtin Name");
+  if (!Postfix.empty())
```

Comment:
I was first confused by this requirement because in SPV IR it is pretty normal that function name has underscores, for example in case SPIR-V instruction may return different types, so the corresponding SPV IR function will have _R{return_type} postfix. But I guess this function is for built-in variables only, so it is ok to expect empty prefix, am I right?

Reply:
Yes, I think you're right. Only use `getSPIRVBuiltin` when you expect builtin variables, otherwise, the content of `Postfix` might be meaningless.